Kernel/Bus/PCI/Access.cpp

```diff
@@ -207,6 +207,12 @@ void disable_memory_space(Address address)
 {
     write16(address, PCI_COMMAND, read16(address, PCI_COMMAND) & ~(1 << 1));
 }
+
+bool is_expnasion_rom_base_address_valid(Address address)
```

Comment:
Typo, should be `is_expansion_rom_base_address_valid`.

Reply:
Oops, thanks for noticing! :)